mathics/eval/numerify.py

```diff
@@ -0,0 +1,71 @@
+# cython: language_level=3
+# -*- coding: utf-8 -*-
+
+"""
+
+Produces a new expression equivalent to the original,
+s.t. inexact numeric elements are reduced to Real numbers with
```

Comment:
Maybe "Produces a new expression with the inexact numeric elements reduced to Real numbers ..."